lib/build/ld

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 set -e
 
-exec ~/.symbolica/build/llvm-link ~/.symbolica/build/musl/libc.bc "$@"
+exec llvm-link-12 ~/.symbolica/build/musl/libc.bc "$@"
```

Comment:
I decided to remove the `llvm-link` indirection script, but let me know if you'd rather keep it still. I know you use these scripts to run things in other, non-docker, envs too, but this one looked like it's only uses were from other scripts in this folder.

Reply:
In fact, I remember why I removed it now. It's because when I build `llvm-link` it's already named `llvm-link` and so we don't need the indirection script anymore for docker.